### ISSUE TITLE
Switch to native date picker for lobby timer?

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#8122]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

Apparently the other one is not accessible so all kind of places migrate to a native one:
* https://github.com/nextcloud/server/pull/33915
